cmake/SetupPackages.cmake

```diff
@@ -118,16 +118,7 @@ if (RAJA_ENABLE_CUDA OR RAJA_ENABLE_HIP OR RAJA_ENABLE_OPENMP OR RAJA_ENABLE_MPI
 endif ()
 
 if (RAJA_NEEDS_BLT_TPLS)
```

Comment:
I think this conditional is now redundant since blt_install_tpl_setups only installs enabled tpls.

Reply:
good point